core/src/main/java/org/pentaho/di/core/row/ValueDataUtil.java

```diff
@@ -853,25 +854,48 @@ public static Long divideLongs( Long a, Long b ) {
     return new Long( a.longValue() / b.longValue() );
   }
 
-  public static BigDecimal divideBigDecimals( BigDecimal a, BigDecimal b, MathContext mc ) {
-    BigDecimal result = null;
+  private static RoundingMode toRoundingMode( String roundingMode ) {
```

Comment:
RoundMode.valueOf( String ) does this all for you I believe.

Reply:
Yes, it does!